lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,14 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cisco5-64' => {
+          'platform' => 'cisco-5-x86_64',
+          'template' => 'cisco-nxos-9k-x86_64'
+        },
+        'cisco7-64' => {
```

Comment:
In hipchat, @LuvCurves mentioned, "we have a production version of cisco-wrlinux-7 now. It is called 'cisco-exr-9k-x86_64'". Does that mean in the future we might have a different production version of `cisco-wrlinux-7` based on Nexus or IOS? If so, then I don't think `cisco7-64` is specific enough. In my mind the beaker-hostgenerator string should identify which vmpooler `template` to use, but not necessarily the `platform`. For example, `centos7-64` maps to the `centos-7-x86_64` template:

```
'centos7-64' => {
  'platform' => 'el-7-x86_64',
  'template' => 'centos-7-x86_64'
},
```

But both `centos7-64` and `redhat7-64` have platform `el-7-x86_64`.